precompiles/types/coin.go

```diff
@@ -16,13 +17,20 @@ func ZRC20ToCosmosDenom(ZRC20Address common.Address) string {
 	return config.ZRC20DenomPrefix + ZRC20Address.String()
 }
 
-func CreateCoinSet(zrc20address common.Address, amount *big.Int) (sdk.Coins, error) {
+func CreateZRC20CoinSet(zrc20address common.Address, amount *big.Int) (sdk.Coins, error) {
 	defer func() {
 		if r := recover(); r != nil {
 			return
 		}
 	}()
 
+	if (zrc20address == common.Address{}) {
+		return nil, &ErrInvalidAddr{
+			Got:    zrc20address.String(),
+			Reason: "empty address",
+		}
+	}
```

Comment:
Please add a unit test for this

Reply:
Added in 2b130fdc0f29a4d958d599d633c42f3987a6e7b5